src/constants/theme/spacings.ts

```diff
@@ -8,5 +8,5 @@ export const SPACINGS = {
   md: `${wp('6%')}px`, // 24px
   lg: `${wp('8%')}px`, // 32px
   xl: `${wp('10%')}px`, // 40px
-  xxl: `${wp('12%')}px`, // 48px
+  xxl: `${wp('2%')}px`, // 48px
```

Comment:
This should not be changed as it affects various other screens/spacings. `xxl` should stay `xxl`. Instead, you should use `xs` instead of `xxl` in the corresponding place.

Reply:
ok, will do that